migration_scripts/README.md

```diff
@@ -0,0 +1,145 @@
+- [Prerequisites](#prerequisites)
+- [Migration Scripts](#migration-scripts)
+- [Backup Content and Config](#backup-content-and-config)
+- [Create Mappings](#create-mappings)
+- [Move files](#move-files)
+- [Create redirects](#create-redirects)
+- [Sidebar Unification](#sidebar-unification)
+- [Check That All Pages Are Linked to in the Sidebar](#check-that-all-pages-are-linked-to-in-the-sidebar)
+- [Position Files and Folders - Ready For Testing](#position-files-and-folders---ready-for-testing)
+- [Testing Changes Locally](#testing-changes-locally)
+- [Push Changes](#push-changes)
+- [Clean Up](#clean-up)
+
+# Prerequisites
+
+```bash
+pip3 install openpyxl
```

Comment:
Thanks Vaughn, 
I will add `requests`.
Perhaps because it is just me running this we don't need to set up a `venv`.
I will make these changes is VS Code and push a new commit in the next hour or so.
Thanks so much the the :eyes: and :bulb: - really appreciate your collaboration on this.